### PR TITLE
[DOCFIX] Checksum and cp command, and a typo

### PR DESCRIPTION
**Pull request by bissont**
[Updated_Commmand_Line_Documentation.pdf](https://github.com/Alluxio/alluxio/files/389695/Updated_Commmand_Line_Documentation.pdf)

### CODE REVIEW

**Comment by gpang** [comment]
LGTM

File: docs/en/Command-Line-Interface.md [hunk 1]
```diff
@@ -141,6 +149,16 @@ data files and their total size for any date, month, or year.
 
 {% include Command-Line-Interface/count.md %}
 
+## cp
+The `cp` command copies a file or directory in the Alluxio filesystem.
+
+If the `-R` option is used and the source designates a directory, cp copies the entire subtree at
+source to the destination.
+
+For example, `cp` can be used to copy files between UnderFS filesystems.
```
Comment (ifcharming) [inline docs/en/Command-Line-Interface.md]: nit : Under file systems, or just UnderFS.

**Comment by ifcharming** [comment]
LGTM, just a minor nit.

File: docs/en/Command-Line-Interface.md (hunk 1 above)
Comment (bissont) [inline docs/en/Command-Line-Interface.md]: Done.

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**